app/src/main/java/com/uc/mymovies/dto/Movie.kt

```diff
@@ -1,7 +1,10 @@
 package com.uc.mymovies.dto
 
-data class Movie(val title: String, val genre: String, val director: String, val image: String, val synopsis: String, var isFavorite: Boolean=false) {
-
-
+data class Movie(val title: String,
+                 val genre: String,
+                 val director: String,
+                 val image: String,
+                 val synopsis: String,
+                 var isFavorite: Boolean=false) {
```

Comment:
This is a purely cosmetic change, and pretty inconsequential overall. When I first opened the project, I couldn't see all of the movie attributes without scrolling to the right on my monitor. Breaking this declaration up into multiple lines makes it more readable.